__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+
+describe("Backend testing", () => {
+  describe("GET /api/topics", () => {
+    test("status:200, responds with an array of topic data", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: { topics } }) => {
+          expect(topics).toBeInstanceOf(Array);
+          topics.forEach((topic) => {
```

Comment:
If you are using a forEach, you will need to test for the length of topics so we know that is going through each object.